diagnostics-app/src/app/core/types/global.types.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/**
+ * @fileoverview Defines types used in global scope by multiple modules.
+ */
+
+export type Theme = 'dark' | 'light';
```

Comment:
IIUC, typeof Theme is still a string. It seems like you are trying to limit the assignable values by doing this which you can achieve better using enums.